docs/Developers/Bear_Installation_Tool.rst

```diff
@@ -1,6 +1,9 @@
 Bear Installation Tool
 ======================
 
+**PLEASE NOTE: cib is currently broken. Most of the commands listed here will
+ not work.**
```

Comment:
can we instead have one of those sphinx/docutils directives like a note/warning?

the syntax is something like `.. warning:` but I am not fully sure